### PR TITLE
Add new methods file

### PR DESCRIPTION
**Pull request by janczer**
Two methods:

- `size`
- `chmod`
- `basename`
- `join`
- `split`

Add tests.

### CODE REVIEW

File: test_fixtures/require_test/file.gb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require("file")
```
Comment (st0012) [inline test_fixtures/require_test/file.gb]: `require_test` means that is for testing `require`
you should put this file in `text_fixtures` and name it `file_test.gb`

Comment (st0012) [inline test_fixtures/require_test/file.gb]: You should also test these methods in separate files. And test them like I test `require_relative`  in [here](https://github.com/goby-lang/goby/blob/master/goby_test.go). You should execute these files and check if the return value is right.

**Comment by st0012** [comment]
@janczer  BTW you can add some sample files under `samples/file_lib` to demonstrate this new package.

File: goby_test.go [hunk 2]
```diff
@@ -18,6 +18,46 @@ func TestRequireRelative(t *testing.T) {
 	testIntegerObject(t, result, 160)
 }
```
Comment (st0012) [inline goby_test.go]: Can we also add size method's test here?

**Comment by st0012** [comment]
Just need one more test then we can merge this 😁

**Comment by janczer** [comment]
@st0012 Add test for `size`

**Comment by st0012** [comment]
@janczer Thanks, can you continue adding more methods like, `open`, `write` that can modify files?

**Comment by janczer** [comment]
@st0012 Yes, of course. I add issue about #115. Please add methods what you think should be in `File`.

**Pull request merged**